docs/content/docs/test/Run.md

```diff
@@ -14,7 +14,7 @@ type: docs
 При создании Pull Request'а ваша ветка ребейзится на целевую ветку и автоматически запускается
 [`fastCheck`](http://links.k.avito.ru/PV), объявленный в `avito/build.gradle`:
 - Юнит тесты
-- Инструментальные тесты:
+- Инструментальные тесты (кроме помеченных аннотацией [`@Flaky`]({{< ref "/docs/test/FlakyTests.md" >}})):
```

Comment:
I'm thinking about renaming `Flaky` to `Unstable`. 
Just a thought after today's case with Google Maps.

Reply:
`Unstable` was one of the first naming variants. But I decided that Flaky is a commonly-known term.